apps/teamsjs-cc-app/readme.md

```diff
@@ -0,0 +1,93 @@
+# TeamsJS Capability Checker
```

Comment:
Can this file have some info about what this app is and what it's for?

Some of the info about how to run it is good and helpful and should stay -- but some details specific to this app would be good.